src/server/api/schema.js

```diff
@@ -1080,11 +1080,15 @@ const rootMutations = {
           // run interaction step handler
           try {
             const handler = require(`../../integrations/action-handlers/${interactionStepAction}.js`);
-            handler.processAction(
+            const handlerPromise = handler.processAction(
```

Comment:
oops, this is `handlerPromise` vs. below it's `handlerResult`.  Let's change one of them to the other!